README.md

```diff
@@ -522,6 +522,8 @@ tasks:
       - name: POD_ID
 ```
 
+New tasks will become available once the stack has updated and refreshed. You can press the `Refresh statuses & commits` button to manually trigger this refresh.
```

Comment:
Is this correct? I thought tasks were loaded from the last successfully deployed commit. Not 100% sure here, but if I'm right then triggering a refresh will not be sufficient.

Reply:
Perhaps I should have write "Once merged to Shipit and deployed, new tasks..."?